corehq/apps/users/forms.py

```diff
@@ -797,9 +797,12 @@ def __init__(self, project, request_user, *args, **kwargs):
                     },
                 '''
             )
-            send_email_field = crispy.Field(
-                'send_account_confirmation_email',
-                data_bind='checked: send_account_confirmation_email, enable: sendConfirmationEmailEnabled',
+            send_email_field = crispy.Div(
+                crispy.Field(
+                    'send_account_confirmation_email',
+                    data_bind='checked: send_account_confirmation_email, enable: sendConfirmationEmailEnabled',
+                ),
+                data_bind='visible: sendConfirmationEmailEnabled'
```

Comment:
nit: would it be easy to change `sendConfirmationEmailEnabled` to be `requireAccountConfirmation` or something similar? I feel like it's currently too similar to `send_account_confirmation_email`, which confused me

Reply:
[6d2f12e](https://github.com/dimagi/commcare-hq/pull/36696/commits/6d2f12e09a5f513c247d96821ae923703684d15d)